xwiki-platform-core/xwiki-platform-ratings/xwiki-platform-ratings-ui/src/test/java/org/xwiki/ratings/RatingsTest.java

```diff
@@ -57,34 +51,23 @@
  * @since 12.10.4
  */
 @XWikiSyntax21ComponentList
-@XHTML10ComponentList
+@HTML50ComponentList
+@ComponentList({
+    ControlCharactersFilter.class
```

Comment:
Also needed for the new (soon to be merge ) PageTest, I wonder if it should be moved to a component list.

Reply:
How come it wasn't needed before with `@XHTML10ComponentList`?

Reply:
I had the same issue but was solving it with a not very good solution : 
https://github.com/xwiki/xwiki-platform/pull/1565/files#diff-51624f0dc03a44f8a6bd8ec98997744821b0db08fd6a43bb3e90829eea9d4ed6L71

Reply:
ok. I don't know enough about ControlCharactersFilter and where it's needed to know whether we should move it to a common list or not. For now let's consider it's good in this test.